softwarereview_author.Rmd

```diff
@@ -11,6 +11,7 @@ This concise guide presents the software peer review process for you as a packag
 - Please consider the best time in your package's development to submit.  Your package should be sufficiently mature so that reviewers are able to review all essential aspects, but keep in mind that review may result in major changes.
     - We strongly suggest submitting your package for review _before_ publishing on CRAN or submitting a software paper describing the package to a journal. Review feedback may result in major improvements and updates to your package, including renaming and breaking changes to functions.
     -   Do not submit your package for review while it or an associated manuscript is also under review at another venue, as this may result in conflicting requests for changes.
+- Please also consider the time and effort needed to respond to reviews (for your package's best!): think about your availability or those of your collaborators in the next weeks and months following a submission. We tend to be flexible with deadlines and submissions can be put on hold but we want to avoid wasting volunteer reviewers' time.
```

Comment:
```suggestion
- Please also consider the time and effort needed to respond to reviews: think about your availability or that of your collaborators in the next weeks and months following a submission. We tend to be flexible with deadlines, and submissions can be put on hold but we want to avoid wasting volunteer reviewers' time.
```